DMX.Core.Api.Tests.Acceptance/APIs/ExternalLabs/ExternalLabApiTests.cs

```diff
@@ -11,7 +11,7 @@
 using WireMock.Server;
 using Xunit;
 
-namespace DMX.Core.Api.Tests.Acceptance.APIs.Labs
+namespace DMX.Core.Api.Tests.Acceptance.APIs.ExternalLabs
```

Comment:
It's `Labs` not `ExternalLabs` right?

Reply:
I updated the tests to reflect what we implemented on the Api. I think that if we were to add a RetrieveAll method to the LabApi, and keep the Labs namespace, then there would be conflicts between the two

Reply:
Alternatively, I could add both the ExternalLabApiTests and LabApiTests to the Labs namespace, but it wouldn't follow the pattern of having a folder per endpoint

Reply:
Let's hold off on this PR until we build the `LabOrchestrationService` because there should be no such a thing as `ExternalLabsController` - that's the whole point of abstraction.